apps/music/js/ui/views/player_view.js

```diff
@@ -228,19 +230,17 @@ var PlayerView = {
         this.album.classList.remove('hidden-cover-share');
       }
     } else {
-      var titleBar = document.getElementById('title-text');
-
-      titleBar.textContent =
-        metadata.title || navigator.mozL10n.get('unknownTitle');
-      titleBar.dataset.l10nId = metadata.title ? '' : 'unknownTitle';
+      TitleBar.changeTitleText(
```

Comment:
Yes, you are correct, the open.html also needs these changes. The easiest/fastest way to test the music open activity is to launch the UI tests app(under dev_apps), then go to the API tab, there is a manual test called "Open Audio", you should be able to test the open activity by tapping one of the three options.